planning/behavior_velocity_planner/src/scene_module/occlusion_spot/grid_utils.cpp

```diff
@@ -247,6 +236,198 @@ void getCornerPositions(
     corner_positions.emplace_back(corner_position);
   }
 }
+
+boost::optional<Polygon2d> generateOcclusionPolygon(
+  const Polygon2d & occupancy_poly, const Point2d & origin, const Point2d & min_theta_pos,
+  const Point2d & max_theta_pos, const double ray_max_length = 100.0)
+{
+  using tier4_autoware_utils::normalizeRadian;
+  const double origin_x = origin.x();
+  const double origin_y = origin.y();
+  const double delay_angle = M_PI / 6.0;
```

Comment:
What is "delay_angle" ? Is it ok to fix it at 30°?

Reply:
@0x126 
It's not concretely defined yet this is the simplest way of avoiding dynamic object occlusion detection.
To define delay below is necessary .
- distance to object
- relative velocity to dynamic object
- relative angle
- estimated perception delay 
But with this info change of struct grid_utils should be changed and not yet tested.

so I remain this for TODO task in another PR.

Reply:
[5988206](https://github.com/autowarefoundation/autoware.universe/pull/620/commits/5988206b0a4e6f514fb7e1775960199816aec632)

@0x126 
I added comment here